src/mongoose_graphql/admin/mongoose_graphql_admin_stanza_mutation.erl

```diff
@@ -0,0 +1,66 @@
+-module(mongoose_graphql_admin_stanza_mutation).
+
+-export([execute/4]).
+
+-ignore_xref([execute/4]).
+
+-include("../mongoose_graphql_types.hrl").
+-include("mongoose_logger.hrl").
+-include("jlib.hrl").
+
+execute(_Ctx, _Obj, <<"sendMessage">>, Opts) ->
+    send_message(Opts);
+execute(_Ctx, _Obj, <<"sendMessageHeadLine">>, Opts) ->
+    send_message_headline(Opts);
+execute(_Ctx, _Obj, <<"sendStanza">>, Opts) ->
+    send_stanza(Opts).
+
+send_message(Opts = #{<<"from">> := From, <<"to">> := To, <<"body">> := Body}) ->
+    Packet = mod_commands:build_message(jid:to_binary(From), jid:to_binary(To), Body),
+    do_routing(From, To, Packet).
+
+send_message_headline(Opts = #{<<"from">> := From, <<"to">> := To}) ->
+    Packet = build_message_with_headline(jid:to_binary(From), jid:to_binary(To), Opts),
+    do_routing(From, To, Packet).
+
+send_stanza(Opts = #{<<"stanza">> := Packet}) ->
+    From = jid:from_binary(exml_query:attr(Packet, <<"from">>)),
+    To = jid:from_binary(exml_query:attr(Packet, <<"to">>)),
+    do_routing(From, To, Packet).
+
+do_routing(From = #jid{lserver = LServer}, To, Packet) ->
+    case mongoose_domain_api:get_domain_host_type(LServer) of
+        {ok, HostType} ->
+            do_routing(HostType, LServer, From, To, Packet);
+        _ ->
+            {error, #{what => unknown_domain, domain => LServer}}
```

Comment:
Maybe we can format this error message in a `mongoose_graphql_error:err` callback?